_posts/2019-08-15-release-1.9.0.md

```diff
@@ -0,0 +1,352 @@
+full checkpoints and incremental checkpoints. More details are available in
+[FLIP-43](https://cwiki.apache.org/confluence/display/FLINK/FLIP-43%3A+State+Processor+API)
+
+
+### Stop-with-Savepoint (FLIP-34)
+
+[Cancelling with a
+savepoint](https://ci.apache.org/projects/flink/flink-docs-stable/ops/state/savepoints.html#operations)
+is a common operation for stopping/restarting, forking or updating Flink jobs.
+However, the existing implementation did not guarantee output persistence to
+external storage systems for exactly-once sinks. To improve the end-to-end
+semantics when stopping a job, Flink 1.9 introduces a new `SUSPEND` mode to
+stop a job with savepoint that is consistent with the emitted data.
+You can suspend a job with Flink’s CLI client as follows:
+
+```
+bin/flink stop -s [:targetDirectory] :jobId
+```
+
+The final job state is set to `FINISHED` on success, allowing
+users to detect failures of the requested operation. 
+
+More details are available in
+[FLIP-34](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=103090212)
+
+
+### Flink WebUI Rework
+
+After a
+[discussion](http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/DISCUSS-Change-underlying-Frontend-Architecture-for-Flink-Web-Dashboard-td24902.html)
+about modernizing the internals of Flink’s WebUI, this component was
+reconstructed using the latest stable version of Angular — basically, a bump
+from Angular 1.x to 7.x. The redesigned version is the default in 1.9.0,
+however there is a link to switch to the old WebUI.
+
+<div class="row"> <div class="col-sm-6"> <span><img class="thumbnail"
+    src="{{site.baseurl}}/img/blog/release-19-web1.png" /></span> </div> <div
+    class="col-sm-6"> <span><img class="thumbnail"
+    src="{{site.baseurl}}/img/blog/release-19-web2.png" /></span> </div>
+    </div>
+
+**Note:** Moving forward, we will not guarantee feature parity for the old
+version of the WebUI.
+
+
+### Preview of the new Blink SQL Query Processor
+
+Following the [donation of
+Blink]({{site.baseurl}}/news/2019/02/13/unified-batch-streaming-blink.html) to
+Apache Flink, the community worked on integrating Blink’s query optimizer and
+runtime for the Table API and SQL. As a first step, we refactored the
+monolithic `flink-table` module into smaller modules
+([FLIP-32](https://cwiki.apache.org/confluence/display/FLINK/FLIP-32%3A+Restructure+flink-table+for+future+contributions)).
+This resulted in a clear separation of and well-defined interfaces between the
+Java and Scala API modules and the optimizer and runtime modules.
+
+<span><img style="width:50%"
+src="{{site.baseurl}}/img/blog/release-19-stack.png" /></span>
+
+Next, we extended Blink’s planner to implement the new optimizer interface
+such that there are now two pluggable query processors to execute Table API
+and SQL statements: the pre-1.9 Flink processor and the new Blink-based query
+processor. The Blink query processor offers better SQL coverage (full TPC-H
```

Comment:
should this sentence _**The Blink query processor offers better SQL coverage**_  become as follows for consistency with the previous sentence?  

_The **Blink-based** query processor offers better SQL coverage_